### DISCUSSION TITLE
To  the Community Members and @github-staff

### DISCUSSION THREAD

**Discussion by aakeshaaka** [community/community#73532] [GitHub Education]
### Hi everyone,

Hi everyone,
Hi Team
I am not getting options to apply for GIthub Teacher Swags and The Previous Swags offer i applied i did not got any response from the side of Github.
Option is Not Available for request of swags
Not Got Response of Previous applied offers
I request github team and @github-staff @samus-aran or @divais to look into this concern

**Comment by ettaboyle** [comment]
Hi @pragy-dl,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. Here is your other duplicate post: https://github.com/orgs/community/discussions/73530

As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by aakeshaaka** [reply]
But What is the response kindly tell @ettaboyle
